desafio-02/phewill/numeros_primos.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
Não vai colocar os seus dados (autor, repl.it link, etc) nesse arquivo? Normalmente, coloca-se como comentários no início do arquivo.